security/uefi/gen_uefi_certs.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+#
+# Simple script that generates a set of keys and certificates for UEFI secure boot
+
+set -e
```

Comment:
Do we need it in the final script?

Reply:
is always good to have this option imo so we stop the execution instantly as a query exits while having a non-zero status

`set -ex`

can be useful as well when it run on the ci so we can look on the logs if needed